CHANGELOG.md

```diff
@@ -7,3 +7,17 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ## Added
 - Basic Skel
 - missing PR template
+
+## [0.0.2] - 2016-08-11
```

Comment:
If we can transfer ownership then maybe we will keep the version the same. I don't have access to our rubygems so someone else really needs to assist here.